### ISSUE TITLE
DockLike Widget Customization Question

### ISSUE THREAD

**Issue by IHuGoWI** [ramensoftware/windows-11-taskbar-styling-guide#122]
I wonder if there is any way to move the widget block to the right corner of the taskbar like in the image:
![Screenshot 2024-10-25 003732](https://github.com/user-attachments/assets/ba80eedd-e48b-4fc3-8f4b-32e3e8b3c3a1)

**Comment by bbmaster123** [comment]
Yes, but with some caveats, see [here](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/71)

A better option might be to turn off `ExtendedModeAEPForTaskbar: 48660958` using [vivetool](https://github.com/bbmaster123/FWFU/blob/main/Vivetool_GUI.md)

hope this helps!
